pkgs/development/python-modules/certbot/default.nix

```diff
@@ -33,7 +34,16 @@ buildPythonPackage rec {
     hash = "sha256-Qee7lUjgliG5fmUWWPm3MzpGJHUF/DXZ08UA6kkWjjk=";
   };
 
-  sourceRoot = "${src.name}/${pname}";
+  patches = [
```

Comment:
Next time just drop the pytest.ini instead. The settings in there just make any new warning an error which is for the developers, not for us.